tests/test_build_html.py

```diff
@@ -953,3 +953,16 @@ def test_jsmath(app, status, warning):
             'e^{ix} = \\cos x + i\\sin x</div>' in content)
     assert '<div class="math">\nn \\in \\mathbb N</div>' in content
     assert '<div class="math">\na + 1 &lt; b</div>' in content
+
+
+@with_app(buildername='html', testroot='html_extra_path')
+def test_html_extra_path(app, status, warning):
+    app.builder.build_all()
+
+    assert (app.outdir / '.htaccess').exists() is True
+    assert (app.outdir / '.htpasswd').exists() is False
+    assert (app.outdir / 'API.html_t').exists() is True
+    assert (app.outdir / 'css/style.css').exists() is True
+    assert (app.outdir / 'rimg.png').exists() is True
+    assert (app.outdir / '_build/index.html').exists() is False
+    assert (app.outdir / 'background.png').exists() is True
```

Comment:
Really? I did not known that. I've believed result of the `pep8` command...
Surely, pep8 guide says `Don't compare boolean values to True or False using == .` and `if greeting is True:` is worse.
https://www.python.org/dev/peps/pep-0008/#programming-recommendations

I want to compare results with boolean values intentionally. It seems what the assertion expects is very clear.
But, at the same time, I don't want to violate the guidelines. So, I'll rewrite these assertions.

Thanks,